contrib/cf-remote/cf_remote/main.py

```diff
@@ -108,34 +108,32 @@ def run_command_with_args(command, args):
             edition=args.edition)
     elif command == "uninstall":
         all_hosts = ((args.hosts or []) + (args.hub or []) + (args.clients or []))
-        commands.uninstall(all_hosts)
+        return commands.uninstall(all_hosts)
     elif command == "packages":
-        commands.packages(tags=args.tags, version=args.version, edition=args.edition)
+        return commands.packages(tags=args.tags, version=args.version, edition=args.edition)
     elif command == "run":
-        commands.run(hosts=args.hosts, raw=args.raw, command=args.remote_command)
+        return commands.run(hosts=args.hosts, raw=args.raw, command=args.remote_command)
     elif command == "sudo":
-        commands.sudo(hosts=args.hosts, raw=args.raw, command=args.remote_command)
+        return commands.sudo(hosts=args.hosts, raw=args.raw, command=args.remote_command)
     elif command == "scp":
-        commands.scp(hosts=args.hosts, files=args.args)
+        return commands.scp(hosts=args.hosts, files=args.args)
     elif command == "spawn":
         if args.list_platforms:
-            commands.list_platforms()
-            return
+            return commands.list_platforms()
         if args.init_config:
-            commands.init_cloud_config()
-            return
+            return commands.init_cloud_config()
         if args.name and "," in args.name:
             user_error("Group --name may not contain commas")
         # else
         if args.role.endswith("s"):
             # role should be singular
             args.role = args.role[:-1]
-        commands.spawn(args.platform, args.count, args.role, args.name)
+        return commands.spawn(args.platform, args.count, args.role, args.name)
     elif command == "destroy":
         group_name = args.name if args.name else None
-        commands.destroy(group_name)
+        return commands.destroy(group_name)
     elif command == "deploy":
-        commands.deploy(args.hub, args.directory)
+        return commands.deploy(args.hub, args.directory)
     else:
         user_error("Unknown command: '{}'".format(command))
```

Comment:
missing `return 1` here

Reply:
not really, `user_error` is `sys.exit`

Reply:
Then it's another `sys.exit()` in place where `return` should be used. The code calling this function might be doing some cleanup before exiting, for example.